### DISCUSSION TITLE
Query on dashboard

### DISCUSSION THREAD

**Discussion by NamaniNiha** [jarkko-hautakorpi/iBeacon-indoor-positioning-demo#22] [General]
I am trying to run dashbaord and I see MQTT connection is not getting done.
Also I cant see the prints added for mqtt connection. Am I missing something.??
Appreciate any support w.r.t this. Thanks

**Comment by jarkko-hautakorpi** [comment]
Try debugging the javascript with chrome dev tools and see that the mosquitto is on and working, use some client like https://github.com/eclipse/paho.mqtt-spy to see the messages.
